### PR TITLE
[DOCFIX] Add limitation for S3 API Doc

### PR DESCRIPTION
**Pull request by HelloHorizon**

### CODE REVIEW

File: docs/en/api/S3-API.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ The following table describes the support status for current Amazon S3 functiona
 {% endfor %}
 </table>
 
+### Limitation
+In Alluxio, we use / as reserved separator, so don't create the S3 directory with /, like
+s3://example-bucket//.
```
Comment (ZhuTopher) [inline docs/en/api/S3-API.md:41]:
```suggestion
In Alluxio, we use `/` as a reserved separator. Therefore any S3 directory with an object named `/` (eg: `s3://example-bucket//`) will conflict and behave incorrectly.
```

**Review by ZhuTopher — CHANGES_REQUESTED** [review]
Does my suggested wording make sense to you? Thanks!

**Comment by HelloHorizon** [comment]
It make sense, thanks @ZhuTopher

**Review by ZhuTopher — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**